### DISCUSSION TITLE
Business teams

### DISCUSSION THREAD

**Discussion by matteeyah** [joemasilotti/railsdevs.com#888] [Ideas]
## Summary

At my company I coordinate hiring engineers, but I also work with other people on vetting the candidates and talking to them. I'd like the ability to have a company "team" and add everyone I work with there, instead of all of us sharing one set of user credentials.

This improves the usability for hiring teams with multiple people, but also allows us to avoid having to share login credentials since that's bad security posture.

**Comment by joemasilotti** [comment]
Thanks for chiming in @matteeyah! I mentioned this in #878, but for transparency, let's chat offline about this. I'll post findings here for dev work to begin.
